docker-compose.yml

```diff
@@ -1,6 +1,13 @@
 version: "3.8"
 
 services:
+  redis:
+    image: "redis"
+    command: redis-server
+    ports:
+      - "6379:6379"
+    volumes:
+      - "redis:/var/lib/redis/data"
```

Comment:
`redis:/data` may be more correct, but this change (alone) also doesn't get things working.